package com.profesional.tasks.Exceptions;public class UserNotFoundException {
}
